server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -425,6 +431,12 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
                         sequencedMessage.expHash1 = this.lastHash;
                     }
 
+                    if (sequencedMessage.type === MessageType.Summarize) {
+                        // note: this is being emitted before it's produced to the deltas topic
+                        // that lets event handlers alter the message if necessary
+                        this.emit("summarizeMessage", sequencedMessage as ISequencedDocumentAugmentedMessage);
```

Comment:
Who is listening to this event? Is the listener currently only added in Push, and we would have to implement that here in OSS later if we are interested in opting in for this feature?

Also, would the listener be responsible for setting `value.operation.serverMetadata?.deliAcked`, which is checked in Scribe?

Reply:
Correct on all accounts.